src/cache/cache_las.c

```diff
@@ -353,6 +353,102 @@ __wt_las_cursor_close(
 	return (ret);
 }
 
+/*
+ * __wt_las_page_skip_locked --
```

Comment:
Are there any changes in these two functions other than movement and adding the function name prefix? The code movement makes detecting any changes and reviewing them hard.

Reply:
No changes other than movement and function name prefixes.